common/scripted_effects/wc_artifacts_scripted_effects.txt

```diff
@@ -0,0 +1,409 @@
+set_prefix_and_suffix_temp_flags_to_artifact_effect = {
+	random_list = { # Prefix
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_1 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_2 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_3 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_4 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_5 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_6 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_7 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_8 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_9 }
+		1 = { set_artifact_flag = temp_artifact_prefix_flag_10 }
+	}
+	random_list = { # Suffix
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_1 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_2 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_3 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_4 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_5 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_6 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_7 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_8 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_9 }
+		1 = { set_artifact_flag = temp_artifact_suffix_flag_10}
+	}
+}
+
+finish_artifact_creation_effect = {
+	set_creation_date = 1.1.1
+	clear_flags_with_prefix = temp_artifact_
```

Comment:
What about adding...
```suggestion
	clear_flags_with_prefix = temp_artifact_
	log = "Added generic artifact: [This.GetName] is its name"
```
To debug artifact names?

Reply:
Done